src/App.jsx

```diff
@@ -8,7 +9,9 @@ export const App = () => (
     <p>Sum of 10 and 0 is 10</p>
     <p>Sum of 0 and 5 is 5</p>
     <p>Sum of 0 and 0 is 0</p>
-    {/* Replace paragraphs with Sum componets */}
-    {/* And remove commented lines :) */}
+    <Sum a={2} b={3} />
+    <Sum a={-5} b={5} />
+    <Sum a={10} />
```

Comment:
```suggestion
    <Sum a={10} />
    <Sum b={5} />
```

![image](https://user-images.githubusercontent.com/70009205/208151691-e273090a-9add-4b85-acb0-1cd5f4a1ef17.png)